recaptcha_enterprise/snippets/pom.xml

```diff
@@ -61,6 +69,12 @@
       <artifactId>google-cloud-recaptchaenterprise</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>recaptcha-password-check-helpers</artifactId>
+      <version>1.0.1</version>
+    </dependency>
+
```

Comment:
nit: do we want to use parent BOM instead of picking specific dependencies?

Reply:
This library is not under libraries-bom artifact. Hence, including the version specifically.